### PR TITLE
Poles

### PR DESCRIPTION
**Pull request by eggrobin**
Store polar α and δ for rotating bodies, this allows for non-positive poles (which is the convention for retrograde planets and satellites), and removes singularities that would occur when defining the rotation of bodies.

### CODE REVIEW

File: physics/oblate_body_body.hpp [hunk 1]
```diff
@@ -110,12 +103,14 @@ not_null<std::unique_ptr<OblateBody<Frame>>> OblateBody<Frame>::ReadFromMessage(
...
     MassiveBody::Parameters const& massive_body_parameters) {
   auto const axis = Vector<double, Frame>::ReadFromMessage(message.axis());
+  auto const axis_spherical_coordinates = axis.coordinates().ToSpherical();
   typename RotatingBody<Frame>::Parameters
       rotating_body_parameters(Length(),
                                Angle(),
                                J2000,
-                               AngularVelocity<Frame>(
-                                   axis.coordinates() * Radian / Second));
+                               Radian / Second,
```
Comment (pleroy) [inline physics/oblate_body_body.hpp]: 1 * Radian / Second

File: physics/rotating_body.hpp [hunk 2]
```diff
@@ -59,6 +66,11 @@ class RotatingBody : public MassiveBody {
   // Returns the radius passed at construction.
   Length mean_radius() const override;
 
+  // Returns the axis passed at construction.
```
Comment (pleroy) [inline physics/rotating_body.hpp]: It's not passed at construction anymore.

File: physics/rotating_body_body.hpp [hunk 3]
```diff
@@ -14,43 +14,70 @@ namespace physics {
...
-  CHECK_NE(angular_velocity_.Norm(), 0.0 * SIUnit<AngularFrequency>())
+      angular_frequency_(angular_frequency),
+      right_ascension_of_pole_(right_ascension_of_pole),
+      declination_of_pole_(declination_of_pole) {
+  CHECK_NE(angular_frequency_, 0.0 * SIUnit<AngularFrequency>())
       << "Rotating body cannot have zero angular velocity";
 }
 
-template<typename Frame>
+template <typename Frame>
```
Comment (pleroy) [inline physics/rotating_body_body.hpp]: Remove extra space.

File: physics/rotating_body_body.hpp [hunk 4]
```diff
@@ -96,14 +127,16 @@ RotatingBody<Frame>::ReadFromMessage(
     serialization::RotatingBody const& message,
     MassiveBody::Parameters const& massive_body_parameters) {
   // For pre-Buffon compatibility, build a point mass.
+  // TODO(phl): pre-Cardano compatibility.
```
Comment (pleroy) [inline physics/rotating_body_body.hpp]: Please open an issues, otherwise we'll forget about this.

File: physics/oblate_body_body.hpp (hunk 1 above)
Comment (eggrobin) [inline physics/oblate_body_body.hpp]: Done.

File: physics/rotating_body.hpp (hunk 2 above)
Comment (eggrobin) [inline physics/rotating_body.hpp]: Done.

File: physics/rotating_body_body.hpp (hunk 3 above)
Comment (eggrobin) [inline physics/rotating_body_body.hpp]: Done.

File: physics/rotating_body_body.hpp (hunk 4 above)
Comment (eggrobin) [inline physics/rotating_body_body.hpp]: #1106.

**Pull request merged**